### PR TITLE
haproxy performance tunning for global maxconn.

### PR DESCRIPTION
**Pull request by nirajdp76**
Increase global maxconn and parameterize existing defaults and mysql setting for performance tunning.

### CODE REVIEW

**Comment by nirajdp76** [comment]
retest

**Comment by ddaskal** [comment]
looks good, thanks for splitting this out Niraj! 👍

File: roles/haproxy/defaults/main.yml [hunk 1]
```diff
@@ -1,5 +1,21 @@
 ---
 haproxy:
-  bufsize: 16384
   stats_group: monitor
- 
+  global:
+    maxconn: 20480
+    bufsize: 16384
+  defaults:
+    timeoute_http_request: 30s
```
Comment (ddaskal) [inline roles/haproxy/defaults/main.yml]: misspelling, please change all 3 occurrences in the PR to "timeout_http_request"

File: roles/haproxy/templates/etc/haproxy/haproxy_openstack.cfg [hunk 2]
```diff
@@ -132,14 +132,14 @@ defaults
...
-  timeout client 300s
-  timeout server 300s
-  timeout connect 1s
-  timeout http-keep-alive 60s
-  timeout http-request 30s
+  timeout client {{ haproxy.defaults.timeout_client }}
+  timeout server {{ haproxy.defaults.timeout_server }}
+  timeout connect {{ haproxy.defaults.timeout_connect }}
+  timeout http-keep-alive {{ haproxy.defaults.timeout_http_keep_alive }}
+  timeout http-request {{ haproxy.defaults.timeoute_http_request }}
```
Comment (ddaskal) [inline roles/haproxy/templates/etc/haproxy/haproxy_openstack.cfg]: misspelling, please change all 3 occurrences in the PR to "timeout_http_request"

File: roles/haproxy/templates/etc/haproxy/haproxy_swift.cfg [hunk 3]
```diff
@@ -13,14 +13,14 @@ defaults
...
-  timeout client 300s
-  timeout server 300s
-  timeout connect 1s
-  timeout http-keep-alive 60s
-  timeout http-request 5s
+  timeout client {{ haproxy.defaults.timeout_client }}
+  timeout server {{ haproxy.defaults.timeout_server }}
+  timeout connect {{ haproxy.defaults.timeout_connect }}
+  timeout http-keep-alive {{ haproxy.defaults.timeout_http_keep_alive }}
+  timeout http-request {{ haproxy.defaults.timeoute_http_request }}
```
Comment (ddaskal) [inline roles/haproxy/templates/etc/haproxy/haproxy_swift.cfg]: misspelling, please change all 3 occurrences in the PR to "timeout_http_request"

**Pull request merged**